curriculum/challenges/english/01-responsive-web-design/css-box-model/part-008.md

```diff
@@ -0,0 +1,69 @@
+---
+id: 60a3e3396c7b40068ad69971
+title: Part 8
+challengeType: 0
+dashedName: part-8
+---
+
+# --description--
+
+Time for CSS.
+
+Even though your `<div>` has no text, it's still treated as a box with content.
+Write a CSS rule that uses the `.canvas` class selector and set its `width` to 500 pixels.
+Here's a CSS rule that sets the width of the class `card` to 300 pixels:
+
+```css
+.card {
+  width: 300px;
+}
+```
+
+# --hints--
+
+Your code should have a `.canvas` selector.
+
+```js
+const hasCanvas = new __helpers.CSSHelp(document).getStyleDeclaration('.canvas');
```

Comment:
Something is not right with how helper works, i.e. `getStyleDeclaration` is returning `undefined` seemingly always. This applies to other tests in this part and further challenges.

Reply:
This looks like the issue described here: https://github.com/freeCodeCamp/freeCodeCamp/pull/42266

Reply:
:+1: I can confirm this. After making change manually and restarting `npm run develop`, everything works.

Reply:
Performed a rebase. Should be fixed.